php/class-wp-cli.php

```diff
@@ -1269,13 +1269,15 @@ public static function get_config( $key = null ) {
 	 * * Prevent halting script execution on error.
 	 * * Capture and return STDOUT, or full details about command execution.
 	 * * Parse JSON output if the command rendered it.
+	 * * Include additional runtime arguments to be used with command.
 	 *
 	 * ```
 	 * $options = array(
 	 *   'return'     => true,   // Return 'STDOUT'; use 'all' for full object.
 	 *   'parse'      => 'json', // Parse captured STDOUT to JSON array.
 	 *   'launch'     => false,  // Reuse the current process.
 	 *   'exit_error' => true,   // Halt script execution on error.
+	 *   'command_args' => array('--skip-themes'), // Additional command arguments to be used with $command
```

Comment:
Alignment is off here.
```suggestion
	 *   'return'       => true,                // Return 'STDOUT'; use 'all' for full object.
	 *   'parse'        => 'json',              // Parse captured STDOUT to JSON array.
	 *   'launch'       => false,               // Reuse the current process.
	 *   'exit_error'   => true,                // Halt script execution on error.
	 *   'command_args' => [ '--skip-themes' ], // Additional arguments to be passed to the $command.
```